atom/common/platform_util_win.cc

```diff
@@ -295,24 +296,29 @@ bool OpenItem(const base::FilePath& full_path) {
     return ui::win::OpenFileViaShell(full_path);
 }
 
-bool OpenExternal(const base::string16& url,
-                  const OpenExternalOptions& options) {
+void OpenExternalOnWorkerThread(const base::string16& url,
+                                const OpenExternalOptions& options) {
+  base::ScopedBlockingCall scoped_blocking_call(base::BlockingType::MAY_BLOCK);
   // Quote the input scheme to be sure that the command does not have
   // parameters unexpected by the external program. This url should already
   // have been escaped.
-  base::string16 escaped_url = L"\"" + url + L"\"";
+  std::string escaped_url = url.spec();
   auto working_dir = options.working_dir.value();
 
   if (reinterpret_cast<ULONG_PTR>(
           ShellExecuteW(nullptr, L"open", escaped_url.c_str(), nullptr,
                         working_dir.empty() ? nullptr : working_dir.c_str(),
                         SW_SHOWNORMAL)) <= 32) {
-    // We fail to execute the call. We could display a message to the user.
-    // TODO(nsylvain): we should also add a dialog to warn on errors. See
-    // bug 1136923.
-    return false;
+    // On failure, it may be good to display a message to the user.
+    // https://crbug.com/727913
+    return;
   }
-  return true;
+}
+
+void OpenExternal(const GURL& url) {
```

Comment:
Use already available `void OpenExternal(const base::string16&, const OpenExternalOptions&, OpenExternalCallback)` method, a new signature is unnecessary.